spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/AnnotatedServiceRegistrationBean.java

```diff
@@ -130,4 +156,52 @@ public AnnotatedServiceRegistrationBean setDecorator(
         this.decorator = decorator;
         return this;
     }
+
+    /**
+     * Returns the exception handlers of the annotated service object.
+     */
+    public Collection<? extends ExceptionHandlerFunction> getExceptionHandlers() {
+        return exceptionHandlers;
+    }
+
+    /**
+     * Sets the exception handlers of the annotated service object.
+     */
+    public AnnotatedServiceRegistrationBean setExceptionHandlers(
+            @NotNull Collection<? extends ExceptionHandlerFunction> exceptionHandlers) {
+        this.exceptionHandlers = exceptionHandlers;
+        return this;
+    }
+
+    /**
+     * Returns the request converters of the annotated service object.
+     */
+    public Collection<? extends RequestConverterFunction> getRequestConverters() {
+        return requestConverters;
+    }
+
+    /**
+     * Sets the request converters of the annotated service object.
+     */
+    public AnnotatedServiceRegistrationBean setRequestConverters(
+            Collection<? extends RequestConverterFunction> requestConverters) {
```

Comment:
I don't know if `@NotNull` is necessary or not. But if it needs, we should add it here as well.

Reply:
No need to add here.

Reply:
Then, don't we have to `requireNonNull()` here?